setup.py

```diff
@@ -37,6 +36,7 @@
         'texar-pytorch',
```

Comment:
I see that we install `texar-pytorch` and `texar`. To my knowledge, we use only `texar-pytorch`. For simplicity, we can remove `texar`